### PR TITLE
improved Okapi tests

### PR DESCRIPTION
**Pull request by following5**
### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code

### CODE REVIEW

File: tests/Modules/Okapi/Integration/Services/ApiSrvTest.php [hunk 1]
```diff
@@ -15,8 +15,8 @@ public function testApisrvStatsMethod()
 
         $response = $client->get('services/apisrv/stats');
 
-        self::assertCount(4, $response);
-        self::assertEquals(1, $response['apps_count']);
+        self::assertGreaterThanOrEqual(4, $response);
```
Comment (following5) [inline tests/Modules/Okapi/Integration/Services/ApiSrvTest.php:18]: number of returned fields may increase

File: tests/Modules/Okapi/Integration/Services/ApiSrvTest.php [hunk 2]
```diff
@@ -15,8 +15,8 @@ public function testApisrvStatsMethod()
...
+        self::assertGreaterThanOrEqual(1, $response['apps_count']);
```
Comment (following5) [inline tests/Modules/Okapi/Integration/Services/ApiSrvTest.php:19]: number of registered apps may increase

File: tests/Modules/Okapi/Integration/Services/ApiSrvTest.php [hunk 3]
```diff
@@ -34,6 +34,6 @@ public function testApiSrvInstallationsMethod()
 
         $response = $client->get('services/apisrv/installations');
 
-        self::assertGreaterThanOrEqual(7, $response);
+        self::assertGreaterThanOrEqual(5, $response);
```
Comment (following5) [inline tests/Modules/Okapi/Integration/Services/ApiSrvTest.php:37]: installations may be shut down or removed for other reasons

File: tests/Modules/Okapi/Integration/Services/AttrsTest.php [hunk 4]
```diff
@@ -26,7 +26,7 @@ public function testAttrsAttributeIndexMethodReturnsAttributes()
 
         $response = $client->get('services/attrs/attribute_index', ['consumer_key' => $this->getConsumerKey()]);
 
-        self::assertCount(77, $response);
+        self::assertGreaterThanOrEqual(77, $response);
```
Comment (following5) [inline tests/Modules/Okapi/Integration/Services/AttrsTest.php:29]: attibutes will be added

File: tests/Modules/Okapi/OkapiCredentialsTrait.php [hunk 5]
```diff
@@ -6,12 +6,20 @@
 {
     public function getOkapiUrl()
     {
-        return getenv('URL') . '/okapi';
+        $url = getenv('OKAPI_LOCAL_URL');
```
Comment (following5) [inline tests/Modules/Okapi/OkapiCredentialsTrait.php:9]: allow to set some unique variable for the test site URL

File: tests/Modules/Okapi/OkapiCredentialsTrait.php [hunk 6]
```diff
@@ -6,12 +6,20 @@
...
+        if (!$url) {
+            $url = getenv('URL');
+        }
+        return $url . '/okapi';
     }
 
     public function getConsumerKey()
     {
-        return 'yT2eV9xhwTuHKWVKxdWZ';
+        $consumer_key = getenv('OKAPI_LOCAL_CONSUMER_KEY');
```
Comment (following5) [inline tests/Modules/Okapi/OkapiCredentialsTrait.php:18]: allow to test with any consumer key

File: tests/Modules/Okapi/Integration/Services/AttrsTest.php (hunk 4 above)
Comment (teiling88) [reply tests/Modules/Okapi/Integration/Services/AttrsTest.php:29]: if this happen I want to realize it.

File: tests/Modules/Okapi/Integration/Services/ApiSrvTest.php (hunk 3 above)
Comment (teiling88) [reply tests/Modules/Okapi/Integration/Services/ApiSrvTest.php:37]: if this happen I want to realize it.

File: tests/Modules/Okapi/Integration/Services/ApiSrvTest.php (hunk 2 above)
Comment (teiling88) [reply tests/Modules/Okapi/Integration/Services/ApiSrvTest.php:19]: if this happen I want to realize it.

File: tests/Modules/Okapi/Integration/Services/ApiSrvTest.php (hunk 1 above)
Comment (teiling88) [reply tests/Modules/Okapi/Integration/Services/ApiSrvTest.php:18]: if this happen I want to realize it.

File: tests/Modules/Okapi/OkapiCredentialsTrait.php (hunk 6 above)
Comment (teiling88) [reply tests/Modules/Okapi/OkapiCredentialsTrait.php:18]: please write $consumerKey

**Review by teiling88 — CHANGES_REQUESTED** [review]
I want to merge your getenv changes. Please remove the assertGreaterThanOrEqual

**Comment by following5** [comment]
I think the tests are useless in this form, because they will detect 99% false positives. Tests should trigger on errors, not on added features or configuration changes.

**Comment by following5** [comment]
Btw, "I want" is no valid argument. ;)

**Pull request closed without merging**